libcloudforensics/aws.py

```diff
@@ -0,0 +1,726 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Library for incident response operations on AWS EC2."""
+import binascii
+import datetime
+import logging
+import os
+import re
+
+import boto3
+
+log = logging.getLogger()
+
+EC2_SERVICE = 'ec2'
+ACCOUNT_SERVICE = 'sts'
+UBUNTU_1804_AMI = 'ami-0013b3aa57f8a4331'
+AWS_PATH = os.path.expanduser('~/.aws/')
+RETRY_MAX = 10
+REGEX_TAG_VALUE = re.compile('^.{1,255}$')
+
+
+def CreateSession():
+  """Create an AWS session API service.
+
+  Returns:
+    boto3.Session: An AWS session object.
+  """
+  # Create a default session with credentials stored in ~/.aws/credentials
+  # https://boto3.amazonaws.com/v1/documentation/api/latest/guide
+  # /configuration.html
+  return boto3.session.Session()
```

Comment:
This wrapper is VERY thin, is it necessary? It seems like we could just call `boto3.session.Session()` directly in the rest of the code?